sorc/build_fv3.sh

```diff
@@ -20,7 +20,11 @@ if [ $target = hera ]; then target=hera.intel ; fi
 if [ $target = orion ]; then target=orion.intel ; fi
 
 cd fv3gfs.fd/
-FV3=$( pwd -P )/FV3
 cd tests/
-./compile.sh "$FV3" "$target" "WW3=Y 32BIT=Y" 1
-mv -f fv3_1.exe ../NEMS/exe/global_fv3gfs.x
+#### Modify to turn on CCPP with four suite 
+#### As of 11/12/2020 The FV3_GFS_v16beta has been tested with C768 cycled run on DELL system
+#./compile.sh "$target" "WW3=Y 32BIT=Y" 1
+# mv -f fv3_1.exe ../NEMS/exe/global_fv3gfs.x
+# mv -f fv3_2.exe ../NEMS/exe/global_fv3gfs_ccpp.x
+./compile.sh "$target" "CCPP=Y 32BIT=Y SUITES=FV3_GFS_v15,FV3_GSD_v0,FV3_GSD_noah,FV3_GFS_v16beta" 2 NO NO
+mv -f fv3_2.exe ../NEMS/exe/global_fv3gfs.x
```

Comment:
Please clean this up, thanks!

Are we now forcing the CCPP build on users? Specially line 29. @arunchawla-NOAA